spring-petclinic-microservices/spring-petclinic-admin-server/pom.xml

```diff
@@ -16,12 +16,16 @@
       </parent>
 
     <properties>
-        <spring-boot-admin.version>2.2.2</spring-boot-admin.version>
         <docker.image.exposed.port>9090</docker.image.exposed.port>
         <docker.image.dockerfile.dir>${basedir}/../docker</docker.image.dockerfile.dir>
     </properties>
 
     <dependencies>
+        <!-- Enable Bootstrap -->
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-bootstrap</artifactId>
+        </dependency>
```

Comment:
line #43, can we delete the version here?
Same for line #48

Reply:
Updated;

---

For more issues about dependency versions;

there are dependency issues in petclinic samples,
let's use the issue https://github.com/Azure/azure-sdk-for-java/issues/26234 to do this